### PR TITLE
Modify indexing examples

### PR DESCRIPTION
**Pull request by amandaJayanetti**

### CODE REVIEW

File: demo/index_filters.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+              "brass (copper, zinc alloy); steel (metal); cardboard; wood (unidentified)",
+              "brass (copper, zinc alloy); steel (metal); wood (unidentified)")
+
+data<-data.frame(ID_NUMBER,ITEM_NAME,TITLE,MATERIALS)
+db<- tempfile(pattern="RXapianEx-")
+id<-c(0)
+indexFields<-list(list(index=2,prefix="S"))
+filterFields<-list(list(index=3,prefix="XM",separator=";"))
+xapian_index(db,data,id,indexFields,filterFields,"en")
+
```
Comment (jaylett) [inline demo/index_filters.R]: You probably didn't mean to have quite so many newlines at the end!

File: demo/index_filters.R [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+library(RXapian)
+
+# preparing arguments for xapian_index()
+ID_NUMBER <-c("1974-100",
+              "1938-371",
+              "1985-438")
```
Comment (jaylett) [inline demo/index_filters.R]: Is it worth pulling these into a separate data file, so you don't need to duplicate them for every example?

**Comment by jaylett** [comment]
Since you're splitting this out into separate examples, it might be a good idea to start each file with a short comment explaining what the example demonstrates.

**Comment by amandaJayanetti** [comment]
Done :)

**Comment by jaylett** [comment]
@amandaJayanetti are you going to add the sample data file to the repo? If not, perhaps add a note to the readme saying where to get it from.

**Comment by amandaJayanetti** [comment]
Sample data file is already added to the repo and it gets installed when
installing RXapian.

**Comment by jaylett** [comment]
Oops — sorry, must have missed that!

**Comment by jaylett** [comment]
Sorry, meant to say that this looks good to me — merge when ready :)

**Pull request merged**